src/python/CMakeLists.txt

```diff
@@ -15,7 +15,10 @@ add_test(NAME pywraps2_test COMMAND
 set_property(TEST pywraps2_test PROPERTY ENVIRONMENT
              "PYTHONPATH=$ENV{PYTHONPATH}:${PROJECT_BINARY_DIR}/python")
 
+set(CMAKE_INSTALL_SITELIBDIR "${Python3_SITELIB}"
```

Comment:
Calling this `CMAKE_` seems a bit strange. I would assume that's only for variables defined by CMake itself.

Can you point me at some CMake Python documentation or a canonical example? It seems no one has used this variable name before.

https://www.google.com/search?q=%22CMAKE_INSTALL_SITELIBDIR%22
https://github.com/search?q=CMAKE_INSTALL_SITELIBDIR

Reply:
Also, what problem are you trying to solve? Use of this variable should also be documented somewhere.

Reply:
I'm trying to do something like there: https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html.

Reply:
@jmr, please review #186 (this should clarify my motivation).

Reply:
It seems like there should be a relevant directory somewhere here:
https://cmake.org/cmake/help/latest/module/FindPython3.html

How do other swig/python/windows/cmake projects do it?